aikau/src/test/resources/testApp/WEB-INF/classes/alfresco/site-webscripts/alfresco/layout/ClassicWindow.get.html.ftl

```diff
@@ -0,0 +1 @@
+<@processJsonModel group="share"/>
```

Comment:
group="share" not needed... again, more legacy copy/paste stuff we'd like to avoid.